wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/CommandRobot.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.wpilibj2.command;
+
+import edu.wpi.first.wpilibj.TimedRobot;
+
+/**
+ * The CommandRobot class extends the {@link TimedRobot} class in order to ease functionality for
+ * Command Based Programming.
+ */
+public class CommandRobot extends TimedRobot {
+  private static final double kSchedulerOffset = 0.005;
```

Comment:
Why is there an offset? RobotPy put ours in RobotPeriodic, which is where I think most examples put theirs... it feels like this could potentially lead to a difference in behavior.

Reply:
@Starlight220 , @rzblue , and I had a conversation over discord about whether an offset should be included or not. The command scheduler call is normally placed in the RobotPeriodic method which is why testing is required to rule out any potential issues.

Reply:
Having the scheduler run at the end of robotPeriodic can result in inconsistent timing if a lot of stuff runs before it. By adding an offset, the scheduler timing should be more consistent.

Reply:
But in a command based robot, aren't most things running on the scheduler anyways?

Reply:
Yes, but there's a lot of other stuff that runs in the TimedRobot loop, and teams can also choose to run code inside periodic methods as well.